text/stacks/0001-stack-package-metadata.md

```diff
@@ -45,6 +45,3 @@ Example:
 ## Unresolved Questions and Bikeshedding
 
 Is there a clearer/more accurate label name we can use? Technically "stack" refers to a pair of images but we're only trying to represent the packages on a single image.
-
-
-{{REMOVE THIS SECTION BEFORE RATIFICATION!}}
```

Comment:
@ryanmoran  Is this entire section intended to be removed? It [seems](https://github.com/paketo-buildpacks/rfcs/blob/main/text/0013-contributing-guidelines.md#unresolved-questions-and-bikeshedding) we haven't been doing that consistently so I wasn't sure if that was still the intent.

@sophiewigmore and @brayanhenao I think we should at least remove these lines for now (cause they look kinda silly) and I can follow up with a PR to remove bikeshedding sections from every RFC if that is desired.

Reply:
good with that @ekcasey